docs/ai_actions/work_with_ai_actions.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Create new AI actions or modify existing ones to work faster and increase creativity.
+---
+
+# Work with AI actions
+
+AI actions define what results are available to editors in AI-enabled areas, such as, for example, the AI Assistant.
+If the AI Actions LTS update has been installed and configured in your application, and you have administrator permissions, you can reconfigure the existing AI actions, and create new ones.
+
+## View AI actions
+
+With administrator permissions, you can view all AI actions configured in the application by navigating to the Admin Panel and selecting **AI actions**.
+
+![AI actions in Admin Panel](img/ai_actions_list.png)
+
+You can narrow down the list of AI actions by filtering it by the status, either Enabled or Disabled, or by the type.
+Out of the box, there are two types of AI Actions present in the system:
+
+- **Refine text** - used primarily in [online editor](create_edit_content_items.md#ai-assistant) for performing text-to-text actions, for example: "Rewrite text in formal tone"
```

Comment:
Same remark as below about "refine text" that is meant for refining text, while text-to-text is more generic. Overall, I'm not sure that text-to-text, image-to-text etc are worth mentioning in the context of a user doc.